metadata-io/src/main/java/com/linkedin/metadata/search/utils/ESUtils.java

```diff
@@ -92,7 +92,8 @@ public static QueryBuilder getQueryBuilderFromCriterionForSearch(@Nonnull Criter
       // TODO(https://github.com/linkedin/datahub-gma/issues/51): support multiple values a field can take without using
       // delimiters like comma. This is a hack to support equals with URN that has a comma in it.
       if (SearchUtils.isUrn(criterion.getValue())) {
-        filters.should(QueryBuilders.matchQuery(criterion.getField(), criterion.getValue().trim()));
+        // Add keyword index for any URN field types.
+        filters.should(QueryBuilders.matchQuery(criterion.getField() + ".keyword", criterion.getValue().trim()));
```

Comment:
I don't think this is the right place to make this fix. Looks like this is to compensate for Urn fields in some indices where dynamic mapping is being used for the urn field. We should instead change those mappings to look like [this](https://github.com/linkedin/datahub/blob/master/metadata-io/src/main/java/com/linkedin/metadata/timeseries/elastic/indexbuilder/MappingsBuilder.java#L34).

Reply:
There are no indices that use dynamic mappings. For urn fields in the normal search indices, we have the following mappings 
```"tags": {
          "type": "text",
          "fields": {
            "keyword": {
              "type": "keyword"
            }
          },
          "analyzer": "urn_component"
        },```
So we need to add .keyword. 

Is this function used in timeseries as well?

Reply:
Yes - that seems to be the problem. The timeseries tests are using this function in a way that differs from the other search assumptions

Reply:
Let's separate out the two. They should not be using the same function.

Reply:
Though one thing is that we are not putting .keyword on non-urn fields as well. Wondering if we should make urns default keyword instead of default text

Reply:
@jjoyce0510  is there any way to ignore case-sensitiveness in datahub now?Can you please help me with it
as i want to ignore case-sensitivity in datahub search requests.
cc @dexter-mh-lee @rslanka